robusta/src/main.rs

```diff
@@ -382,61 +403,69 @@ async fn run_game_loop(mut recv: Receiver<InputMessage>, state: SharedState) {
-                            info!("Got message: {}", msg);
-                        }
-                        ClientMessage::JoinTeam { team_id } => {
-                            let Some(client) = state.client_mut(id) else {
-                                warn!("Client {} not found", id);
-                                continue;
-                            };
-                            client.team_id = team_id;
-                        }
-                        ClientMessage::EmbarkTrain { train_id } => {
-                            if let Some(team) = state.team_mut_by_client_id(id) {
-                                team.on_train = Some(train_id);
+                            ClientMessage::Message(msg) => {
+                                info!("Got message: {}", msg);
                             }
-                        }
-                        ClientMessage::DisembarkTrain => {
-                            if let Some(team) = state.team_mut_by_client_id(id) {
-                                team.on_train = None;
+                            ClientMessage::JoinTeam { team_id } => {
+                                let Some(client) = state.client_mut(id) else {
+                                    warn!("Client {} not found", id);
+                                    continue;
+                                };
+                                client.team_id = team_id;
+                            }
+                            ClientMessage::EmbarkTrain { train_id } => {
+                                if let Some(team) = state.team_mut_by_client_id(id) {
+                                    team.on_train = Some(train_id);
+                                }
+                            }
+                            ClientMessage::DisembarkTrain => {
+                                if let Some(team) = state.team_mut_by_client_id(id) {
+                                    team.on_train = None;
+                                }
                             }
                         }
                     }
-                }
-                InputMessage::Server(ServerMessage::Departures(deps)) => {
-                    departures = deps;
-                }
-                InputMessage::Server(ServerMessage::ClientDisconnected(id)) => {
-                    info!("Client {} disconnected", id);
-                    state.connections.retain(|x| x.id != id);
+                    InputMessage::Server(ServerMessage::Departures(deps)) => {
+                        departures = deps;
+                    }
+                    InputMessage::Server(ServerMessage::ClientDisconnected(id)) => {
+                        info!("Client {} disconnected", id);
+                        state.connections.retain(|x| x.id != id);
+                    }
                 }
             }
+            PLAYERS_ONLINE.store(!state.connections.is_empty(), std::sync::atomic::Ordering::SeqCst);
```

Comment:
```suggestion
            PLAYERS_ONLINE.store(!state.connections.is_empty(), std::sync::atomic::Ordering::Relaxed);
```
Relaxed should be enough here, since we're not using the atomic to synchronize anything.